src/stores/DaoStore.ts

```diff
@@ -66,16 +66,16 @@ export default class DaoStore {
         });
       });
     });
-    Object.keys(unparsedCache.proposals).map(proposalId => {
+    Object.keys(unparsedCache.proposals).forEach(proposalId => {
       unparsedCache.proposals[proposalId].values = unparsedCache.proposals[
         proposalId
       ].values.map(value => {
         return bnum(value);
       });
       unparsedCache.proposals[proposalId].stateInScheme =
-        unparsedCache.proposals[proposalId].stateInScheme;
+        unparsedCache.proposals[proposalId].stateInScheme; // eslint-disable-line no-self-assign

```

Comment:
I think these two lines do nothing and are safe to remove. (Or maybe I'm missing something obvious.)

Reply:
Removed!